<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopJCDialog</class>
 <widget class="QWidget" name="TopJCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="PaintWidget" name="paintWidget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>501</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="loginfo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hello World&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="onlinePeerView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header>gui/paintwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
